meta-lmp-base/recipes-connectivity/bluez5/bluez5_5.65.bbappend

```diff
@@ -0,0 +1,16 @@
+# This enables using BSD library libedit for client/mesh features
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+READLINE_LIB = "${@bb.utils.contains('LMP_DISABLE_GPLV3', '1', 'libedit', 'readline', d)}"
+
+PACKAGECONFIG:append = " client ${READLINE_LIB} "
```

Comment:
@ricardosalveti 

> Is readline only required when client is enabled?

readline/libedit libraries also use ncurses/curses/termcap, they are selected by autoconf.

>  let autoconf abort in case a readline compatible library is not selected.

I'm sorry, I don't seem to get it. Do you mean to let READLINE_LIB be undefined? In this case, the default configure option will be --with-readline=auto, and if we have a readline library available, it will be selected as more priority. Do you mean this?

Reply:
Just thinking how to best depend on the right readline library (in a way we can also send upstream, without depending on LMP_DISABLE_GPLV3). It seems that readline is only required in bluez when the client functionality is enabled, which is why we might just let it use auto and abort if not found (which seems to be what your patch is doing), then we can have only:

```
PACKAGECONFIG[readline] = "--with-readline=readline,,readline,"
PACKAGECONFIG[libedit] = "--with-readline=libedit,,libedit,"
PACKAGECONFIG[client] = "--enable-client,--disable-client"
```

Reply:
Done.